app/services/analytics_events.rb

```diff
@@ -84,6 +84,14 @@ def account_visit
     track_event('Account Page Visited')
   end
 
+  # @identity.idp.event_name Add Email: Email Confirmation
+  # @param [Boolean] success
```

Comment:
Should this have been excluded?

```
[warn]: @param tag has unknown parameter name: success 
    in file `app/services/analytics_events.rb' near line 91
```

```suggestion
```

Reply:
https://github.com/18F/identity-idp/pull/6084#discussion_r830271411

I should have added it as a kwarg as well as documenting it

Reply:
https://github.com/18F/identity-idp/pull/6109